client/transactions/list/index.js

```diff
@@ -375,6 +403,17 @@ export const TransactionsList = ( props ) => {
 						}
 						autocompleter={ autocompleter }
 					/>,
+					<Button
```

Comment:
We're always displaying this button, even if the table has 0 results. What do you think of adding a check similar to [this one](https://github.com/woocommerce/woocommerce-admin/blob/cc0c881a60948c5961b31a2b46eb73339777bbcb/client/analytics/components/report-table/index.js#L428) to hide the button when the table is empty?

Reply:
💯 Agree, thanks for pointing this out!